templates/python-functionapp-to-linux-on-azure.yml

```diff
@@ -0,0 +1,82 @@
+# Python Function App to Linux on Azure
+# Build a Python function app and deploy it to Azure as a Linux function app.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
+
+trigger:
+- master
+
+variables:
+
+  # Azure Resource Manager connection created during pipeline creation
+  azureSubscription: '{{ azureRmConnection.Id }}'
+
+  # Function app name
+  functionAppName: '{{ functionAppName }}'
+
+  # Agent VM image name
+  vmImageName: 'ubuntu-latest'
+
+stages:
+- stage: Build
+  displayName: Build stage
+  jobs:  
+  - job: Build
+    displayName: Build
+    pool:
+      vmImage: $(vmImageName)
+      
+    steps:
+
+    - bash: |
+        if [ -f extensions.csproj ]
+        then
+            dotnet build extensions.csproj --runtime ubuntu.16.04-x64 --output ./bin
+        fi
+      displayName: 'Build extensions'
+
+    - task: UsePythonVersion@0
+      displayName: 'Use Python 3.6'
+      inputs:
+        versionSpec: 3.6
+
+    - bash: |
+        python3.6 -m venv worker_venv
+        source worker_venv/bin/activate
+        pip3.6 install setuptools
```

Comment:
I don't think we need to install the setuptools here. When creating the virtual environment, it should contain the setuptools by default.

Reply:
@Hazhzeng is there something we changed recently ? This was needed when last time we did build designer templates about 4 months back.  
If you can confirm at your end that these are not needed, we will remove them from both the templates and save some build time

Reply:
Yes, the virtual environment created by python3.6 venv already has setuptools inside. It is safe to remove `pip3.6 install setuptools`.

Reply:
Done, thanks